src/client/components/ConsentsSubHeader.tsx

```diff
@@ -158,6 +159,7 @@ const pageProgression = css`
 const ul = css`
   display: flex;
   list-style: none;
+  justify-content: space-between;
```

Comment:
We use the flex rule `space-between` here to spread each item out evenly across the span of the progress step container